src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,26 @@
 const accessibilityQuiz = [
+  {
+    Question: "What is the purpose of the 'aria-labelledby' attribute in ARIA?",
+    Answer:
+      "To enable authors to reference other elements on the page to define an accessible name.",
+    Distractor1: "To hide an element from screen readers",
+    Distractor2: "To add a tooltip to an element",
+    Distractor3: "To change the focus order of elements",
+    Explanation:
+      "The 'aria-labelledby' attribute specifies the element or elements responsible for providing labels to the element to which it is assigned.",
+    Link: "https://forum.freecodecamp.org/t/why-is-aria-labelledby-needed/589633/17"
```

Comment:
Please do not use forum posts that are not posted by the official camperbot freeCodeCamp account.
Reason being is because these posts can be deleted at any time.
You should use posts from the freeCodeCamp news publication, freeCodeCamp YouTube, MDN or W3.org

```suggestion
    Link: "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby"
```